lib/database/model/sthDatabaseNameMapperTool.js

```diff
@@ -389,27 +389,54 @@ function mapOrUnmapCollection(options, databaseName, collectionName, callback) {
  */
 function copyDatabase(options, databaseName, callback) {
```

Comment:
```suggestion
// Alternative function without copydb command
function copyDatabase(options, databaseName, callback) {
```

In order to do in the same way the other place.

Actually, instead of repeating twice the same exact code (is the same, isn't it?) it would be better to refactor copyDatabase() to a common place.

Reply:
They are different, because one is used to encode and the other one to map, which are different methods to choose the name of the new DB. Furthermore, the information passed to ´sthLogger.info´ is different in each function

Reply:
Ok, thus NTC.